app/scripts/views/settings.js

```diff
@@ -179,7 +179,11 @@ define(function (require, exports, module) {
     },
 
     _setupAvatarChangeLinks () {
-      this.$('.avatar-wrapper > *').wrap('<a href="/settings/avatar/change" class="change-avatar"></a>');
+      if (this.supportsAvatarUpload()) {
+        this.$('.avatar-wrapper > *').wrap('<a href="/settings/avatar/change" class="change-avatar"></a>');
+      } else {
+        this.$('.avatar-wrapper').addClass('nohover');
```

Comment:
@vladikoff @shane-tomlinson There might be a better way to do this but couldn't find one in reasonable amount of time. This adds a no hover class when in iOS 10 and css only applies hover state when no hover class is specified. Tried

 * using `removeClass`
 * unbinding `mouseenter mouseleave` events